Commands/poll.py

```diff
@@ -0,0 +1,115 @@
+
+        #check if there is a question
+        if '?' not in input_str:
+            await ctx.send(translate("poll_no_questionmark", await culture(ctx)))
+            return
+
+        input_split = input_str.split('?',1)
+        question = input_split[0]
+        numbers_and_options = input_split[1].strip()
+        
+        # parse timeout
+        first_word = numbers_and_options.split()[0]
+        if not first_word.isdigit():
+            await ctx.send(translate("poll_no_timeout", await culture(ctx)).format(POLL_DEFAULT_TIMEOUT))
+            timeout_s = POLL_DEFAULT_TIMEOUT * 60
+        else:
+            timeout_s = int(first_word) * 60
+        
+        if timeout_s > POLL_MAX_TIMEOUT * 60:
+            await ctx.send(translate("poll_max_timeout", await culture(ctx)).format(POLL_MAX_TIMEOUT))
+            timeout_s = POLL_MAX_TIMEOUT * 60
+
+        # parse options
+        options = numbers_and_options.split(first_word,1)[1].strip()
+        if len(options) > 0:
+            options_list = options.split(';')
+            is_yes_no = False
+        else:
+            is_yes_no = True
+        
+        # create message to send to channel
+        txt = translate("poll_start", await culture(ctx)).format(poller_id,question)
+
+        # add options to message
+        options_dict = dict()
+        if is_yes_no:
+            txt += translate("poll_yes_no", await culture(ctx)).format(yes,no)
+            options_dict[yes] = translate("yes", await culture(ctx))
+            options_dict[no] = translate("no", await culture(ctx))
+        else:
+            i = 1
+            for option in options_list:
+                txt += "{} - {}\n".format(number_emojis[i],option)
+                options_dict[number_emojis[i]] = option
+                i += 1
+        
+        msg = await ctx.send(txt)
+
+        # add reactions to message
+        if is_yes_no:
+            await msg.add_reaction(yes)
+            await msg.add_reaction(no)
+        else:
+            i = 1
+            for option in options_list:
+                await msg.add_reaction(number_emojis[i])
+                i += 1
+
+        # wait until timeout
+        await asyncio.sleep(timeout_s)
+
+        # refresh message
+        msg = await ctx.fetch_message(msg.id)
+        
+        # get the reactions
+        reactions = msg.reactions
+        reactions_dict = dict()
+        for reaction in reactions:
+            reactions_dict[reaction.emoji] = reaction.count
+        reactions_sorted = sorted(reactions_dict.items(), key=lambda x: x[1], reverse=True)
+
+        # send poll results
+        txt = translate("poll_results", await culture(ctx)).format(drum,poller_id,question)
+        for reaction in reactions_sorted:
+            try:
+                option_str = options_dict[reaction[0]].strip()
+                count = reaction[1] - 1
+                txt += translate("poll_votes", await culture(ctx)).format(option_str,count)
+            except:
```

Comment:
I think it is considered bad practice to catch app errors, it would be better to specify the type of error you are expecting (such as ValueError etc)